<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关注微信公众号：学点编程--简单浏览器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pb_new">
          <property name="text">
           <string>新页面</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/new.png</normaloff>:/icon/res/new.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_home">
          <property name="text">
           <string>主页</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/home.png</normaloff>:/icon/res/home.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_forward">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>前进</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/forward.ico</normaloff>:/icon/res/forward.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_back">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>后退</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/back.ico</normaloff>:/icon/res/back.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_refresh">
          <property name="text">
           <string>刷新</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/reload.ico</normaloff>:/icon/res/reload.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_stop">
          <property name="text">
           <string>停止</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/res/stop.ico</normaloff>:/icon/res/stop.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_go">
        <property name="text">
         <string>GO</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/res/go.png</normaloff>:/icon/res/go.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
